Hide currently selected episode in 'More From Season' section

jmqm opened the pull request:
**Changes**
Filter out currently selected episode in 'More From Season' section that's visible in an episode's detailed view.

Before
![image](https://github.com/jellyfin/jellyfin-web/assets/68612932/c6738957-c08c-4db9-b4d0-d987f48b77e0)

After
![image](https://github.com/jellyfin/jellyfin-web/assets/68612932/3990f020-3195-4bd8-a7fb-5d32ccfa204b)

thornbill commented on src/controllers/itemDetails/index.js:
```diff
@@ -1095,7 +1095,7 @@ function renderMoreFromSeason(view, item, apiClient) {
             section.querySelector('h2').innerText = globalize.translate('MoreFromValue', item.SeasonName);
             const itemsContainer = section.querySelector('.itemsContainer');
-            cardBuilder.buildCards(result.Items, {
+            cardBuilder.buildCards(result.Items.filter((i) => i.Id !== item.Id), {
```
Let's move the filtering to be at the start of the function in the `.then()`.

```ts
const items = result.Items.filter(i => i.Id !== item.Id);
```

The if statement checking the length will need updated also.

thornbill commented:
@jmqm did you mean to close this?

jmqm commented:
Yes. Implemented your suggestion but I noticed a problem.

Originally, if I'm on episode 12/12 for example, `More From Season` scrolls to the episode. When I removed the currently-selected episode from `More From Season`, it stopped doing that.

I was debugging it for a while and eventually I didn't want to continue further. I was working on an additional feature, to blur unwatched episodes, but it was too hectic and causing problems on other areas. I lost interest in both, the codebase is too unorganized for me, and didn't want to continue.

Apologies for the time wasted.
